templates/index.html

```diff
@@ -67,6 +67,24 @@ <h3>{{ contributor.name }}</h3>
             </div>
           </div>
           {% endfor %}
+          <div class="item">
```

Comment:
I assume this exists as a "buffer" for the last element? Would there be any way to deduplicate it?

Reply:
Yep. I think it's possible to deduplicate with some additional JS code, but this one is the safest and simplest way.

Reply:
It is quite a lot of duplication, and pretty simple to resolve with JS- I think it is worth changing.
`scroller.appendChild(scroller.firstElementChild.cloneNode(true));` works